ops/docker-compose-side.yml

```diff
@@ -0,0 +1,191 @@
+# # Accounts and their usage
```

Comment:
I've separated these services into a separate docker compose file so that they're not taking resources during integration tests - they're not essential.